message AppendRaftEntriesResponse {
    required int32 term = 1;
    required bool success = 2;
}